draft-irtf-cfrg-frost.md

```diff
@@ -261,6 +261,8 @@ We now detail a number of member functions that can be invoked on `G`.
 - DeserializeScalar(buf): Attempts to map a byte array `buf` to a `Scalar` `s`.
   This function can raise a DeserializeError if deserialization fails; see
   {{ciphersuites}} for group-specific input validation steps.
+- IdentifierToScalar(i): Maps an identifier `i` to a Scalar with the same numeric
```

Comment:
We might want to be specific that this is the output of `encode_u16()`: a 2-byte,
  big-endian byte string

Reply:
I had in mind that the identifier is the _input_  to `encode_u16()` (i.e. "An identifier is an integer value associated with a participant, or signer, and is a value in the range [1, MAX_SIGNERS]."), which is only encoded when needed (like a Scalar or Element)

Reply:
Perhaps that indicates an Identifier type (which is secretly a Scalar?)